homeassistant/components/media_player/plex.py

```diff
@@ -225,7 +259,7 @@ def state(self):
             elif state == 'paused':
                 return STATE_PAUSED
         # This is nasty. Need to find a way to determine alive
-        elif self.device:
+        elif self.identifier:
```

Comment:
Plex could never be off in the current version either, self.device always existed...